modules/junit-jupiter/src/test/java/org/testcontainers/junit/jupiter/MySqlContainerIT.java

```diff
@@ -0,0 +1,24 @@
+package org.testcontainers.junit.jupiter;
+
+import org.junit.jupiter.api.Test;
+import org.testcontainers.containers.MySQLContainer;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * This test verifies, that setup and cleanup of containers works correctly.
```

Comment:
I wonder right know, if this comment is still true for the existing Spock tests structure. Either way, this is kind of hackish and I think we should refactor this test in the Spock module and don't want to duplicate it for this new module :D